Sources/XMTPiOS/Client.swift

```diff
@@ -100,23 +101,40 @@ public final class Client: Sendable {
 	}
 
 	static func create(account: SigningKey, apiClient: ApiClient, options: ClientOptions? = nil) async throws -> Client {
-		let privateKeyBundleV1 = try await loadOrCreateKeys(for: account, apiClient: apiClient, options: options)
+		let (privateKeyBundleV1, source) = try await loadOrCreateKeys(for: account, apiClient: apiClient, options: options)
+
+		let dbURL = URL.documentsDirectory.appendingPathComponent("xmtp-\(account.address).db3")
+		let v3Client = try await LibXMTP.createClient(
```

Comment:
V3 creation should not happen unless a flag is set in the ClientOptions and the environment is Local. Be good to safe guard that.

Reply:
Added in 04a95aa83977578b32aa14030fed3518748fb83f.